app/presenters/setting_presenter.rb

```diff
@@ -22,6 +22,26 @@ def self.model_name
     Setting.model_name
   end
 
+  # explicit value from mass assignment can not be nil
+  def _assign_attribute(k, v)
```

Comment:
Is this overriding some built in from model?

Reply:
I think it's this: https://github.com/rails/rails/blob/2715b5db76e78620a018269e7cc41330ba6752c0/activemodel/lib/active_model/attribute_assignment.rb#L47-L54

Reply:
Should this method also be private, just like ActiveModel?

Reply:
Yes, it is. The internal method of ActiveModel, I do not have a better solution for distinguishing between mass assignment and direct assignment.